src/helpers/index.ts

```diff
@@ -6,6 +6,7 @@ import {extractEnumsInfo, extractEnumsInfoSafe} from './enumExtractor';
 import * as gitHelper from './git';
 import * as shell from './shell';
 import * as telemetryHelper from './telemetry';
+import UriHandlerController from './uriHandlerController';
```

Comment:
If it was up to me, I would remove this `import`/`export` overhead, and `import` directly in `extension.ts`. The rationale is just the less overhead, the simpler is to understand the dependency flow. Thoughts?